docs/Developers/Development_Setup.rst

```diff
@@ -83,25 +83,18 @@ master branch from and all of its dependencies with pip using
 
 ::
 
+    (coala-venv)$ # For the coala repository:
     (coala-venv)$ git clone https://github.com/coala/coala
-    (coala-venv)$ cd coala
-    (coala-venv)$ pip3 install -e .
-    (coala-venv)$ cd -
+    (coala-venv)$ pip3 install -e coala
+    (coala-venv)$ # For the coala-bears repository:
     (coala-venv)$ git clone https://github.com/coala/coala-bears
-    (coala-venv)$ cd coala-bears
-    (coala-venv)$ pip3 install -e .
+    (coala-venv)$ pip3 install -e coala-bears
 
-Then you can install a repository-backed version of the repository you would
-like to modify using
 
-::
-
-    (coala-venv)$ pip3 install -e <path/to/clone>
```

Comment:
In the [issue](https://github.com/coala/coala/issues/3903) it says it should be removed.
Would it be ok to write a note below the coala-specific commands for the general command, like this:
```
...
(coala-venv)$ git clone https://github.com/coala/coala-bears
(coala-venv)$ pip3 install -e coala-bears
```
To generally install a repository-backed version you can use
```
pip3 install -e <path/to/clone>
```